pkg/kubefed2/disable.go

```diff
@@ -62,17 +62,13 @@ var (
 
 type disableType struct {
 	options.GlobalSubcommandOptions
-	disableTypeOptions
```

Comment:
As per my previous comment, I would prefer that the options set by flag remain in the type for consistency with the other kubefed commands.  We can revisit this for all commands in the future.

Reply:
OK, I'll revert the latest commit